### PR TITLE
fix: Remove ArtworkSubmissionMessage session clue from clue list

### PR DESCRIPTION
**Pull request by olerichter00**
### Description

This PR removes the `ArtworkSubmissionMessage` session clue from the clue list to prevent the message from being shown when the app starts (see [docs about session clues](https://github.com/artsy/eigen/blob/main/docs/add_a_visual_clue.md#L63)). 

I really think we shouldn't use the logic and data structure of visual clues for these messages. It gets super confusing. 

### PR Checklist

- [ ] I tested my changes on **iOS** / **Android**.
- [ ] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

File: src/app/Scenes/MyCollection/MyCollection.tsx [hunk 1]
```diff
@@ -97,29 +97,29 @@ const MyCollection: React.FC<{
...
+    if (!relay.hasMore()) {
+      notifyMyCollectionInsightsTab()
+    }
 
-    notifyMyCollectionInsightsTab()
-  }, [])
+    console.log("MyCollection.tsx: useEffect: relay.hasMore()", artworks.length)
+
+    relay.loadMore(100)
+  }, [me?.myCollectionConnection])
```
Comment (olerichter00) [inline src/app/Scenes/MyCollection/MyCollection.tsx:116]: These changes are from https://github.com/artsy/eigen/pull/6860, just mixed up the base of this PR.

**Review by MrSltun — DISMISSED** [review]
Great work Ole!! 🙌 🚀

File: src/app/Scenes/MyCollection/MyCollection.tsx [hunk 2]
```diff
@@ -97,29 +97,29 @@ const MyCollection: React.FC<{
...
+  // Load all artworks and then check whether all of them have insights
   useEffect(() => {
-    relay.loadMore(100)
+    if (!relay.hasMore()) {
+      notifyMyCollectionInsightsTab()
+    }
 
-    notifyMyCollectionInsightsTab()
-  }, [])
+    console.log("MyCollection.tsx: useEffect: relay.hasMore()", artworks.length)
```
Comment (MrSltun) [inline src/app/Scenes/MyCollection/MyCollection.tsx:113]: 👀

**Review by MrSltun — APPROVED** [review]
I believe now it shouldn't have a conflict :)
Thank you for the solution Ole!! 🙌

**Pull request merged**